rocketpy/simulation/flight.py

```diff
@@ -1181,10 +1241,60 @@ def __init_equations_of_motion(self):
         if self.equations_of_motion == "solid_propulsion":
             self.u_dot_generalized = self.u_dot
 
-    def __init_equations_of_motion(self):
-        """Initialize equations of motion."""
-        if self.equations_of_motion == "solid_propulsion":
-            self.u_dot_generalized = self.u_dot
+    def __init_controllers(self):
+        """Initialize controllers and sensors"""
+        self._controllers = self.rocket._controllers[:]
+        self.sensors = self.rocket.sensors.get_components()
```

Comment:
We are really missing a `Rocket.get_controllers()` method, acessing private variables from other classes is not a good practice.

```suggestion
        self._controllers = self.rocket.get_controllers()
        self.sensors = self.rocket.sensors.get_components()
```

Reply:
I prefer leaving the private attribute for now just so we can change it later when adding complete controllers

Reply:
Just to be clear, I'm not suggesting changing the private attribute to a public one. Indeed I prefer if it stay private.

The problem is to access the private attribute directly. We should be using a `get_......` method.

Just keep this in mind for the future.